src/linker/Linker.Steps/BlacklistStep.cs

```diff
@@ -126,16 +126,14 @@ static ResolveFromXmlStep GetResolveStep (string descriptor)
 
 		protected static XPathDocument GetExternalDescriptor (EmbeddedResource resource)
 		{
-			using (var sr = new StreamReader (resource.GetResourceStream ())) {
-				return new XPathDocument (sr);
-			}
+			using var sr = new StreamReader (resource.GetResourceStream ());
```

Comment:
Is this syntax really better?

Reply:
I agree.  I'm not a big fan of the bracketless using